packages/components/package.json

```diff
@@ -82,6 +80,10 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.1.0",
     "@rollup/plugin-typescript": "^11.1.2",
+    "@storybook/addon-actions": "^7.3.1",
+    "@storybook/types": "^7.3.1",
+    "@testing-library/jest-dom": "^6.0.0",
+    "@types/testing-library__jest-dom": "^6.0.0",
```

Comment:
These look sus 👀 They should probably go in the root package.json?

Reply:
Ah these were here to fix a type issue, but I just removed them and everything is fine

Reply:
So this was the problem:
https://github.com/cultureamp/kaizen-design-system/actions/runs/5899311603/job/16001624423?pr=3992#step:4:3097

`pnpm` seems much stricter than `yarn` when it comes to workspaces, in that root level packages aren't "shared"